alcs-frontend/src/app/services/planning-review/planning-review.service.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../environments/environment';
+import { ToastService } from '../toast/toast.service';
+import { CreatePlanningReviewDto, PlanningReviewDto } from './planning-review.dto';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PlanningReviewService {
+  private url = `${environment.apiUrl}/planning-review`;
+
+  constructor(private http: HttpClient, private toastService: ToastService) {}
+
+  async create(meeting: CreatePlanningReviewDto) {
+    try {
+      return await firstValueFrom(this.http.post<PlanningReviewDto>(`${this.url}`, meeting));
+    } catch (err) {
+      this.toastService.showErrorToast('Failed to create planning review');
```

Comment:
Do we want to add console.log(err) to try catch block so it will be easier to debug code or we want to keep UI as clean as possible?

Reply:
Sounds good to me! Added.